README.md

```diff
@@ -1,2 +1,307 @@
+
+```
+<program_dir>/woltka/tests/data
+```
+
+One can execute the following commands to make sure that Woltka functions correctly, and to get an impression of the basic usage of Woltka.
+
+gOTU table generation:
+
+```bash
+woltka gotu -i align/bowtie2 -o table.tsv
+```
+
+Taxonomic profiling at the ranks of phylum, genus and species as defined in NCBI taxdump:
+
+```bash
+woltka classify \
+  -i align/bowtie2 \
+  --nodes taxonomy/nodes.dmp \
+  --names taxonomy/names.dmp \
+  --rank phylum,genus,species \
+  -o output_dir
+```
+
+Functional profiling by UniRef entries then by GO terms (molecular process):
+
+```bash
+woltka classify \
+  -i align/bowtie2 \
+  --coords function/coords.txt.xz \
+  --map function/uniref.map.xz \
+  --map function/go/process.tsv.xz \
+  --map-is-rank \
+  --rank uniref,process \
+  -o output_dir
+```
+
+
+## gOTU analysis
+
+The Woltka interface provides two commands:
+
+- **gotu**: gOTU table generation.
+- **classify**: Complete classification workflow with all parameters.
+
+The command specialized for gOTU analysis, **gotu**, is a simplified wrapper of the full-power command, **classify**, which has more parameters to allow for extended features and controls. We will start with this simple analysis before diving into the complex and flexible usages.
+
+### Background
+
+The notion of “gOTU” (pronounced as "go-to") is the minimal unit for community ecology studies based on shotgun metagenome or other forms of whole-genome microbiome data. It is in constrast to conventional practices, in which taxonomic units such as genera or species were used. Therefore,  gOTU is analogous to sOTU in 16S rRNA studies. The advantage of using gOTU includes 1) highest-possible resolution, 2) independent from taxonomy which is coarse and error-prone as a classification system. 3) allowing for phylogeny-based analysis such as Faith’s PD and UniFrac. The last part is enhanced by the “web of life” reference phylogeny.
+
+### gOTU table generation
+
+To generate a gOTU table, one needs a multiplexed alignment file, or a directory of per-sample alignment files. These files can be generated by aligning sequencing data against a reference genome database. We recommend using [**SHOGUN**](https://github.com/knights-lab/SHOGUN) with the "Web of Life" database (**WoL**, available for download at: https://biocore.github.io/wol/). For example:
+
+```bash
+shogun align -a bowtie2 -d WoLr1 -i input.fasta -o .
+```
+
+Then one can run Woltka to convert the alignment file(s) into a gOTU table:
+
+```bash
+woltka gotu -i alignment.bowtie2.sam -o table.tsv
+```
+
+The output file `table.tsv` is a table with rows as genome IDs (gOTUs), columns as sample IDs, and cell values as counts of gOTUs in samples.
+
+Note: Both SHOGUN and WoL are available at the [**Qiita**](https://qiita.ucsd.edu/) server. If you are a Qiita user, the alignment file can be automatically generated and downloaded from the Qiita interface.
```

Comment:
Can you provide explicit links?

Reply:
Explicit links added elsewhere for the database files. But Qiita command doesn't have an explicit URL. I may add a screenshot or something.

Reply:
Okay, if that is the case, then more explicit directions would be great! I kind of had to search around a bit.

Reply:
Added a markdown page `app.md` to explain usage of third-party tools, databases and servers, including Qiita. Commit [42a6a46](https://github.com/qiyunzhu/woltka/pull/1/commits/42a6a46bcf33cb270d398eb758f1cd423789d9dd).